parlai/scripts/self_chat.py

```diff
@@ -60,6 +60,12 @@ def setup_args(parser=None):
         choices=['conversations', 'parlai'],
         help='Format to save logs in. conversations is a jsonl format, parlai is a text format.',
     )
+    parser.add_argument(
+        '-p',
```

Comment:
let's make this a little more descriptive, like `--partner-model-file` or `-pmf`

Reply:
--partner-model-file seems good!